SwrveSDK/src/amazon/AndroidManifest.xml

```diff
@@ -0,0 +1,17 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.swrve.sdk"
+    android:versionCode="2"
+    android:versionName="2.8">
+
+    <uses-sdk
+        android:minSdkVersion="8"
+        android:targetSdkVersion="22" />
```

Comment:
Simply based on google AndroidManifest.xml in this case: 
https://github.com/Swrve/swrve-android-sdk/blob/release-4_7/SwrveSDK/src/google/AndroidManifest.xml#L8